DEVELOPMENT.md

```diff
@@ -70,12 +70,40 @@ _These tests will not run correctly unless you have [checked out your fork into
 The integration tests live in [`integration`](./integration) and can be run with:
 
 ```shell
+export GCS_BUCKET="gs://<your bucket>"
+export IMAGE_REPO="gcr.io/somerepo"
 make integration-test
 ```
 
-_These tests require push access to a project in GCP, and so can only be run
-by maintainers who have access. These tests will be kicked off by [reviewers](#reviews)
-for submitted PRs._
+You can also run tests with `go test`, for example to run tests individually:
+
+```shell
+export GCS_BUCKET="gs://<your bucket>"
+export IMAGE_REPO="gcr.io/somerepo"
+go test -v --bucket $GCS_BUCKET --repo $IMAGE_REPO -run TestLayers/test_layer_Dockerfile_test_copy_bucket
```

Comment:
Since we have the flags here anyway, it might be easier to just pass in the names instead of having env variables as an extra step?

Reply:
kk sounds good!

Reply:
Okay so I'm going to do something halfway to what you are suggesting, just removing the export lines so it looks like this:
```
go test -v --bucket $GCS_BUCKET --repo $IMAGE_REPO -run TestLayers/test_layer_Dockerfile_test_copy_bucket`
```

One alternative would be to hardcode some defaults, but i think this makes it hard for ppl to know what they need to change:

```
go test -v --bucket gs://kaniko-test-bucket --repo gcr.io/kaniko-test -run TestLayers/test_layer_Dockerfile_test_copy_bucket`
```

Or we could do something like this, which is what I used to do all the time, but the thing is you can't copy and paste it:
```
go test -v --bucket <your bucket> --repo <your repo> -run TestLayers/test_layer_Dockerfile_test_copy_bucket`
```

I like leaving the env vars in b/c then if ppl use env vars they can copy & paste the command from the doc

Reply:
For sure, SGTM!